Source/SelfService/Web/microservice/purchaseOrder/erpIcon.tsx

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+import React from 'react';
+import clsx from 'clsx';
+
+import logoInfor from '../../images/infor.png'; // with import
+import logoIFS from '../../images/ifs.png';
+import logoSAP from '../../images/sap.png';
+
+import { makeStyles, Theme, createStyles } from '@material-ui/core/styles';
+
+type Props = {
+    kind: 'infor' | 'ifs' | 'sap';
+    onClick: any;
```

Comment:
```suggestion
    onClick: (event: MouseEvent<HTMLImageElement>) => void;
```

I did some digging to find the correct type for the `onClick`, [this example](https://felixgerschau.com/react-typescript-onclick-event-type/) uses `Button`, but we got an `Image` so I just changed that and tested it out locally and it works all fine.